src/__tests__/searchBar.test.tsx

```diff
@@ -1,10 +1,10 @@
 import { render, fireEvent } from '@testing-library/react'
-import SearchBar from '@/components/SearchBar.js'
-import data from '../../data/university_data'
+import SearchBar from '@/components/SearchBar'
+import data from '../../data/university_data.json'
```

Comment:
we will ask @bacpactech to either fetch the data from the query for testing or can we hardcode some colleges.

Reply:
I guess we can use mockData for the while